src/Engine/MapController.java

```diff
@@ -435,6 +445,28 @@ public void onStateChange()
       case LEAVE_MAP:
         // Handled as a special case in handleGameInput().
         break;
+      case SAVE:
+        String filename = "save/" + myGame.saveFile; // "svp" for "SaVe Peace"
```

Comment:
This comment might be more cromulent in `GameInstance.getSaveName()`.

Reply:
Fixed in `moldyInfo`